draft-ietf-avtcore-rtp-over-quic.md

```diff
@@ -1152,8 +1154,7 @@ This section provides a list of items that an application might want to export
 from an underlying QUIC implementation. It is thus RECOMMENDED that a QUIC
 implementation exports the listed items to the application.
 
-* *Maximum Datagram Size*: The maximum datagram size that the QUIC connection
-  can transmit.
+* *Maximum Datagram Size*: The maximum datagram size that the QUIC connection can transmit. Since datagram frames cannot be fragmented, the QUIC implementation MUST also provide a way to query the maximum datagram size so that an application can create RTP packets that always fit into a QUIC datagram frame.
```

Comment:
I would prefer to keep the normative MUST out of here. #138 removes all the other normative SHOULDs from this section, too. I would suggest removing the MUST here and in the datagram encapsulation section. Instead, we should explain why the application should be able to learn about the maximum size and what can happen if it cannot. If the application does not know the maximum size, it may try to send datagrams that will be dropped. I think it makes sense to explain that in the datagram encapsulation section.

Reply:
> I would prefer to keep the normative MUST out of here. #138 removes all the other normative SHOULDs from this section, too. I would suggest removing the MUST here and in the datagram encapsulation section. Instead, we should explain why the application should be able to learn about the maximum size and what can happen if it cannot. If the application does not know the maximum size, it may try to send datagrams that will be dropped. I think it makes sense to explain that in the datagram encapsulation section.

@mengelbart - that makes sense to me, and I'll make the change you're suggesting, but I should point out that 

> This section provides a list of items that an application might want to export
from an underlying QUIC implementation. It is thus **RECOMMENDED** that a QUIC
implementation exports the listed items to the application.

is also BCP14-normative ... our terminology section uses the BCP14 template, and [BCP14 uses SHOULD and RECOMMENDED as synonyms](https://datatracker.ietf.org/doc/html/rfc2119#autoid-3). 

Would you like me to make a change there, too?

Reply:
No need to. It will already be changed with #138.